packages/react-native-reanimated/src/culori/lrgb.ts

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+/*
+ * The vast majority of the code exported by this module is a direct copy of the code from
+ * the culori package (see https://culorijs.org/), which deserves full credit for it.
+ */
+
```

Comment:
Are the functions from the same file? Maybe we can put a link there to related files in a `culori` library for easier access

Reply:
The file is a merge of several files; I'll include a ref to their common directory